src/main/java/com/coreos/jetcd/exception/ConnectException.java

```diff
@@ -4,7 +4,7 @@
  * Signals that an error occurred while attempting to connect to
  * etcd.
  */
-public class ConnectException extends Exception {
+public class ConnectException extends RuntimeException {
```

Comment:
no public constructor.
`ConnectException (String cause, Throwable throwable) {}`

use EtcdExceptionFactory to create this exception.
```java
// jetcd/exception/EtcdExceptionFactory.java
ConnectException newConnectException (String cause, Throwable throwable) {
return new ConnectException (cause, throwable );
}
```